dusty/preflight.py

```diff
@@ -73,14 +89,18 @@ def _ensure_config_dir_exists():
 
 def preflight_check():
     logging.info('Starting preflight check')
+    errors = []
+    errors.append(_check_nginx())
+    errors.append(_check_rsync())
+    errors.append(_check_virtualbox())
+    errors.append(_check_boot2docker())
+    errors.append(_check_docker())
+    errors.append(_check_docker_compose())
+    errors.append(_assert_hosts_file_is_writable())
```

Comment:
Get thunky with it

```
for check in [_check_nginx, ...]:
    errors.append(check())
```

You can take advantage of them all having the same call signature.